apps/macos/src/Podfile

```diff
@@ -28,6 +28,7 @@ use_test_app! do |target|
     pod 'FluentUI-React-Native-Button', :path => '../../../packages/experimental/NativeButton/FluentUIReactNativeButton.podspec'
     pod 'FluentUI-React-Native-Avatar', :path => '../../../packages/experimental/Avatar/FluentUIReactNativeAvatar.podspec'
     pod 'RNCPicker', :path => '../../../node_modules/@react-native-picker/picker'
+    pod 'RNSVG', :path => '../../../node_modules/@microsoft/react-native-svg-desktop'
```

Comment:
Do we need to update the iOS podfile as well? I think iOS should use the same library since FluentTester pulls in 'react-native-svg-desktop' not 'react-native-svg'.